src/openforms/submissions/models/submission_value_variable.py

```diff
@@ -133,6 +137,19 @@ def remove_variables(self, keys: list) -> None:
             if key in self._variables:
                 del self._variables[key]
 
+    # TODO: Argument request present for when we will use the data in the session to put
+    #  data returned from DigiD/eHerkenning into static vars
+    def static_data(self, request: "Request") -> dict:
+        if self._static_data is None:
+            static_form_variables = self.submission.form.formvariable_set.filter(
+                source=FormVariableSources.static
+            )
```

Comment:
I'm wondering - if we are not saving the values in the DB, is there even any point in storing these with the form in the DB?

* we are populating the API endpoint from a hardcoded python list of static variables
* we only do this so the form designer can load that information to include them in the form designer UI
* the form designer includes them in the API call
* we do some "tricky" things during import/copy etc. to ensure that the static variables are present

I think we can ditch static variables being persisted in the DB (even at form-design time) completely and only keep the definition in python code. Advantages:

1. We don't have to query them, ever -> free performance
2. We don't have to deal with new/updated variables in data migrations when we make changes to them in newer versions of OF -> easier upgrade path
3. Related to above - existing forms get the static variables "for free" when upgrading to 1.2.x
4. We can still present them in the UI from the API endpoint we already have, just don't save them in the backend